.github/workflows/rti-docker.yml

```diff
@@ -5,6 +5,8 @@ on:
     branches:
       - master
   workflow_dispatch:
+  workflow_call:
+
```

Comment:
I think that for this to work, the `checkout` action needs to be given a  `repository: lf-lang/lingua-franca` parameter (or it will checkout the calling repo).

Reply:
Updated!